### DISCUSSION TITLE
Column  unsigned attribute invalid?

### DISCUSSION THREAD

**Discussion by hiscaler** [cycle/orm#456] [Q&A]
```
    #[Column(type: 'tinyInteger', default: 24, unsigned: true)]
    private int $time_limit = 24;
```

SQL is
```
 ALTER TABLE `user`
                    CHANGE `time_limit` `time_limit` tinyint(4) NOT NULL DEFAULT 24
```

unsigned is invalid

**Comment by roxblnfk** [comment]
Should be valid when you use MySQL. In other cases it will be ignored

**Reply by hiscaler** [reply]
I use MySQL, but invalid

**Reply by roxblnfk** [reply]
@msmakouz need to add a checking for that `unsigned` option https://github.com/cycle/annotated/blob/af04e98ce87abb14afaa8d65a1fda7b790fc41bc/tests/Annotated/Unit/Attribute/ColumnTest.php#L13
or mb it's issue for migrator

**Reply by msmakouz** [reply]
@roxblnfk @hiscaler Hi, a problem was found in the cycle/migrations, they ignore attributes. PR: https://github.com/cycle/migrations/pull/58
Tests have been added to cycle/annotated, no problems were found. PR: https://github.com/cycle/annotated/pull/94

**Reply by roxblnfk** [reply]
Hi @hiscaler
The issue was fixed in the latest release. Thank you for feedback

**Reply by hiscaler** [reply]
Thank you!
